### PR TITLE
LL parser support to create protocol API review from autorest yaml 

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Integrate parser so APIView can take yaml as input file to generate protocol reviews.

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
@LibbaLawrence @kristapratico @lmazuel : This PR has changes to integrate protocol parser in APIView,

**Review by pakrym — APPROVED** [review]

**Comment by praveenkuttappan** [comment]
@pakrym That was so quick. Thanks for the approval.

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml [hunk 1]
```diff
@@ -184,10 +184,7 @@
                             <div class="tab-pane" id="protocol-help">
                                 <ol>
                                     <li>
-                                        Generate json file using Protocol parser
-                                    </li>
-                                    <li>
-                                        Upload the resulting json file.
+                                        Upload generated yml file from autorest
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:187]: QQ, why don't we upload an autorest readme md instead?

Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:187]: And run autorest as part of the generation?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:187]: That's the next step. These are interim solution while Libba is working on parser. Our final goal is to generate yml file within APIView itself.

Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:187]: or have a UI where you can point to one or multiple spec URLs

File: packages/python-packages/protocol-stub-generator/protocol/parse_yml.py [hunk 2]
```diff
@@ -78,13 +78,14 @@ def parse_yaml(self):
...
         # Create main_view for LLC API View
         main_view = ProtocolClientView.from_yaml(data)
-
-        # Write output to .json
-        j = open(OUTPUT_FILE, "w")
         parsed_json = json.dumps(main_view.to_json(), default=lambda o: o.__dict__)
-        j.write(parsed_json)
 
-        j.close()
+        # Write output to .json
```
Comment (praveenkuttappan) [inline packages/python-packages/protocol-stub-generator/protocol/parse_yml.py:83]: @LibbaLawrence @kristapratico Can you please review this change? I have puled changes from Libba's PR with some minor changes.

**Review by kristapratico — APPROVED** [review]

**Review by LibbaLawrence — APPROVED** [review]

**Review by LibbaLawrence — APPROVED** [review]

**Pull request merged**